README.md

```diff
@@ -12,8 +12,8 @@ Metrics package.
 * A unique UUID v4 random identifier is generated according to [RFC4122][RFC4122]
 * The screen width and height
 * The version of Atom being used
-* The name of each item opened in a pane such as `EditorView`, `SettingsView`,
-  and `MarkdownPreviewView`
+* The name of each item opened in a pane, e.g. `EditorView`, `SettingsView`,
```

Comment:
I would rephrase this to something more like "The name of each view class opened in a pane, e.g. ..."